Improve contributing (#509)
socketsupply/socket: CONTRIBUTING.md

## Contribution Guide Overview
This guide provides steps on how to make contributions, report bugs, and become a part of the ongoing development of `Socket Runtime`.  
If you are just started learning about Socket Runtime, you can start from [here](https://socketsupply.co/guides/).  

## Setup and build the framework

To contribute to the project you need to clone the main repository first and install the dependencies. 

### MacOS and Linux

```bash
git clone git@github.com:socketsupply/socket.git
cd socket 
./bin/install.sh
```

### Windows

> ```bash

Author: corrected

## Contribution Guide Overview
This guide provides steps on how to make contributions, report bugs, and become a part of the ongoing development of `Socket Runtime`.  

> If you are just started learning about Socket Runtime, you can start from [here](https://socketsupply.co/guides/).  

Reviewer: ```suggestion
If you are new to the Socket Runtime project, the [guide](https://socketsupply.co/guides/) is a good place start.  
```

> ## Setup and build the framework

Reviewer: ```suggestion
## Requirements
```
Changing the header order is fine, but I think "Requirements" is more idiomatic.

> To contribute to the project you need to clone the main repository first and install the dependencies. 

Reviewer: ```suggestion
To contribute to the project you need to clone the repository and install the dependencies. 
```
"main" and "first" read redundant.
